tests/utils/test_search_utils.py

```diff
@@ -172,11 +299,32 @@ def test_error_comparison_clauses(filter_string, error_message):
         ("tags.acc = \"LR'", "Invalid clause(s) in filter string"),
         ("tags.acc = = 'LR'", "Invalid clause(s) in filter string"),
         ("attribute.status IS 'RUNNING'", "Invalid clause(s) in filter string"),
+        ("status IS 'RUNNING'", "Invalid clause(s) in filter string"),
     ],
 )
-def test_bad_quotes(filter_string, error_message):
+def test_bad_quotes_for_run(filter_string, error_message):
     with pytest.raises(MlflowException) as e:
-        SearchUtils.parse_search_filter(filter_string)
+        SearchUtils.parse_filter_for_run(filter_string)
+    assert error_message in e.value.message
```

Comment:
Hmm, this is out of the scope of this PR, but we should use the `match` argument for `pytest.raises` ([ref](https://docs.pytest.org/en/stable/reference.html#pytest-raises)) instead of `assert error_message in e.value.message`.